readyapi4j-groovy-dsl/src/test/groovy/com/smartbear/readyapi4j/dsl/execution/RecipeExecutionTest.groovy

```diff
@@ -0,0 +1,26 @@
+package com.smartbear.readyapi4j.dsl.execution
+
+import com.smartbear.readyapi4j.execution.Execution
+import com.smartbear.readyapi4j.execution.RecipeExecutor
+
+import static com.smartbear.readyapi4j.dsl.execution.RecipeExecution.executeRecipe
+import static com.smartbear.readyapi4j.dsl.execution.RecipeExecution.executeRecipeLocally
+
+class RecipeExecutionTest extends GroovyTestCase {
+
+    void testThrowsExceptionWhenLocalRecipeExecutorNotInClasspath() throws Exception {
+        shouldFail(IllegalStateException.class, {
+            executeRecipeLocally {
+                get 'http://google.com'
```

Comment:
This will make the test slow and possibly flaky if the internet connection is bad. I suggest that you replace it with a groovy script test ste